docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: '3'
```

Comment:
what is this for? we don't need to run mongo in docker, we can run it as a service, this is what I do on my machine